website/content/en/preview/reference/cloudformation.md

```diff
@@ -213,6 +214,7 @@ actions on `fleet`, `instance`, `volume`, `network-interface`, `launch-template`
   "Condition": {
     "StringEquals": {
       "aws:RequestTag/kubernetes.io/cluster/${ClusterName}": "owned",
+      "aws:RequestTag/eks:eks-cluster-name": "${ClusterName}"
```

Comment:
Same comments here. Does this change anything about what we specify in the sections that we have above?